lms/static/js/spec/main.js

```diff
@@ -776,11 +773,18 @@
         testFiles[i] = '/base/' + testFiles[i];
     }
 
-    // Jasmine has a global stack for creating a tree of specs. We need to load
-    // spec files one by one, otherwise some end up getting nested under others.
-    window.requireSerial(testFiles, function () {
-        // start test run, once Require.js is done
-        window.__karma__.start();
-    });
+    var specHelpers = [
+        'common/js/spec_helpers/jasmine-extensions',
+        'common/js/spec_helpers/jasmine-stealth',
+        'common/js/spec_helpers/jasmine-waituntil'
+    ];
 
+    require(specHelpers, function() {
```

Comment:
can't we just prepend spec helper files to testFiles? They are getting load serially after all.